addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -31,8 +38,13 @@ def action_send_sms(self):
         if record:
             # Returns a proper error if there is a syntax error with qweb
             body = self.env['mail.render.mixin']._render_template(body, self.mailing_id.mailing_model_real, record.ids)[record.id]
+        # Include the unsubscribe link if sms_allow_unsubscribe is enabled
+        if self.mailing_id.sms_allow_unsubscribe:
+            trace_code = self.env['mailing.trace']._get_random_code()
+            unsubscribe_url = self._get_unsubscribe_url(self.mailing_id.id, trace_code)
+            body = '%s\n%s' % (body, _('STOP SMS: %s', unsubscribe_url))
 
-        new_sms_messages_sudo = self.env['sms.sms'].sudo().create([{'body': body, 'number': number} for number in sanitized_numbers])
+        new_sms_messages_sudo = self.env['sms.sms'].sudo().create([{'body': body, 'number': number} for number in numbers])
```

Comment:
why this change?

Reply:
We're receiving "False" in the sanitized number when the number doesn't meet the condition, though the variable is only intended for displaying the number used so we may not need it to be sanitized.